hw/top_englishbreakfast/data/top_englishbreakfast.hjson

```diff
@@ -186,13 +186,28 @@
       // the ip.hjson will declare the clock port names
       // If none are defined at ip.hjson, clk_i is used by default
       clock_srcs: {clk_i: "io_div4"},
-
+      clock_group: "peri",
       // reset connections defines the port to top level reset connection
       // the ip.hjson will declare the reset port names
       // If none are defined at ip.hjson, rst_ni is used by default
-      reset_connections: {rst_ni: "sys_io_div4"},
+      reset_connections: {rst_ni: "lc_io_div4"},
```

Comment:
Why are we now using `lc_io_div4` instead of `sys_io_div4`?

Reply:
`rst_sys` resets the entire system where as `rst_lc` resets the system excluding rv_dm and a small portion of pinmux.  I noted that the same change was done a while ago for Earlgrey (but then we forgot to update English Breakfast in sync).